diff --git a/dist/client/theme-default/composables/sidebar.js b/dist/client/theme-default/composables/sidebar.js
index 9447a885913b5279b3d72e72fb8036713a9a1e2b..118cefef97b4685a1f85351e7fd455d42dbfde0a 100644
--- a/dist/client/theme-default/composables/sidebar.js
+++ b/dist/client/theme-default/composables/sidebar.js
@@ -18,7 +18,7 @@ export function useSidebar() {
             sidebar.value = _sidebar.value;
     });
     const hasSidebar = computed(() => {
-        return (frontmatter.value.sidebar !== false &&
+        return (frontmatter.value.sidebar == true &&
             sidebar.value.length > 0 &&
             frontmatter.value.layout !== 'home');
     });
